Core/Core/Features/Profile/SideMenu/SideMenuView.swift

```diff
@@ -35,7 +35,7 @@ public struct SideMenuView: View, ScreenViewTrackable {
                     Divider()
                     SideMenuMainSection(enrollment)
                     Divider()
-                    if !environment.k5.isK5Enabled, enrollment == .observer {
+                    if !environment.k5.isK5Enabled, enrollment != .observer {
```

Comment:
This will cause the dark mode switch to appear in the student and teacher apps. This section can be removed along with the SideMenuOptionsSection file.